core/deployment/src/main/java/io/quarkus/deployment/dev/IsolatedDevModeMain.java

```diff
@@ -145,6 +146,16 @@ public void accept(String args) {
                 RuntimeUpdatesProcessor.INSTANCE.setConfiguredInstrumentationEnabled(
                         runner.getConfigValue("quarkus.live-reload.instrumentation", Boolean.class).orElse(false));
                 firstStartCompleted = true;
+
+                for (DevModeListener listener : ServiceLoader.load(DevModeListener.class)) {
```

Comment:
Naive question: we don't want to use build items for that?

Reply:
They aren't available at this point. We might be able to make it work, but it's probably more work than it's worth.